content/docs/support/release/_index.md

```diff
@@ -10,7 +10,7 @@ Release notes for Container Storage Modules:
 
 [CSI Drivers](../../csidriver/release)
 
-[CSM for Authorization](../../authorization/v1.x-ga/release)
+[CSM for Authorization](../../authorization/v1.x/release)
```

Comment:
We need to add release notes for Auth 2.x too.

Reply:
fixed

Reply:
added